includes/Modules/Idea_Hub.php

```diff
@@ -730,4 +746,37 @@ private function is_idea_post( $post_id ) {
 		return is_array( $this->get_post_idea( $post_id ) );
 	}
 
+	/**
+	 * Adds .googlesitekit-idea-hub__draft class to idea posts on the posts page.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $classes An array of post class names.
+	 * @param array $class An array of additional class names added to the post.
+	 * @param int   $post_id The post ID.
+	 * @return array An array of post class names.
+	 */
+	private function update_post_classes( $classes, $class, $post_id ) {
+		global $post_type;
+
+		$current_screen = get_current_screen();
```

Comment:
Is this function always defined on front end page loads? I feel like it might not be sometimes.

Reply:
Updated to make sure it runs only on in the admin.

Reply:
Yes of course - what I mean is that `get_current_screen` isn't normally loaded on the front where this filter is also invoked. I see you've added a new guard for the frontend here so this is all good now 👍